### DISCUSSION TITLE
Error: failed to get area dimension...

### DISCUSSION THREAD

**Discussion by SolindekDev** [replit/kaboom#523] [Q&A]
Hi i have a problem when i create an object with area and solid and add into it a onclick event  it's not works and show this error:
![obraz](https://user-images.githubusercontent.com/83218819/152996148-e836fb1a-7dfd-404b-8f08-0654031379fe.png)
![obraz](https://user-images.githubusercontent.com/83218819/152995929-e71b36f1-4f3b-413b-8e50-620b0e9f572f.png)
![obraz](https://user-images.githubusercontent.com/83218819/152995953-462b1ce1-9050-490c-a442-350e716fffd4.png)

Please help...

**Comment by slmjkdbtl** [comment]
Currently `circle()` doesn't support automatic area, you can manually specify the area size 
```js
add([
	circle(30),
	area({ width: 60, height: 60, offset: vec2(-30) }),
])
```

But it will get supported next version

**Reply by SolindekDev** [reply]
ok thanks a lot!
